code/ship/shiphit.cpp

```diff
@@ -514,7 +514,11 @@ float do_subobj_hit_stuff(object *ship_objp, object *other_obj, vec3d *hitpos, i
 #endif
 
 	if (!global_damage) {
-		create_subsys_debris(ship_objp, hitpos);
+		auto submodel = ship_get_subsys_for_submodel(ship_p, submodel_num);
```

Comment:
Why not use `subsys` for the variable name instead of `submodel`?  (`subsys` is used later in the diff.)

Reply:
Thanks, that name was clearly wrong.

Also, I noticed that the condition after this was also wrong so I fixed that as well.